portfolio/src/main/webapp/script.js

```diff
@@ -86,3 +95,40 @@ function jobsBack() {
   setContent();
 }
 
+/**
+ * Change accent color to selected color
+ */
+function changeAccentColor(selectedButton) {
+  var classes = selectedButton.className.split(' ');
```

Comment:
use let or const instead of var 

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let